airbyte-integrations/connectors/source-airtable/source_airtable/source.py

```diff
@@ -43,7 +42,7 @@ def discover(self, logger: AirbyteLogger, config) -> AirbyteCatalog:
 
         Retrieve: Bases, Tables from each Base, generate JSON Schema for each table.
         """
-        auth = TokenAuthenticator(token=config["api_key"])
+        auth = self._auth if self._auth else AirtableAuth(config)
```

Comment:
You can use more simple and readable variant:
`auth = self._auth or AirtableAuth(config)`